diff --git a/setup.py b/setup.py
index 2ba5154..da8ac16 100644
--- a/setup.py
+++ b/setup.py
@@ -56,14 +56,7 @@ def use_pkg_config():
     _ask_pkg_config(extra_link_args,    '--libs-only-other')
     _ask_pkg_config(libraries,          '--libs-only-l', '-l')
 
-if isWindows or isMac:
-    cmake_build = os.path.abspath('external/lensfun/build')
-    install_dir = os.path.join(cmake_build, 'install')
-    
-    include_dirs += [os.path.join(install_dir, 'include', 'lensfun')]
-    library_dirs += [os.path.join(install_dir, 'lib')]
-else:
-    use_pkg_config()
+use_pkg_config()
     
 # this must be after use_pkg_config()!
 include_dirs += [numpy.get_include()]
@@ -205,7 +198,7 @@ package_data = {'lensfunpy': []}
 # evil hack, check cmd line for relevant commands
 # custom cmdclasses didn't work out in this case
 cmdline = ''.join(sys.argv[1:])
-needsCompile = any(s in cmdline for s in ['install', 'bdist', 'build_ext', 'wheel', 'nosetests'])
+needsCompile = False
 if isWindows and needsCompile:
     windows_lensfun_compile()
     package_data['lensfunpy'].append('*.dll')
